android/app/build.gradle.kts

```
@@ -47,11 +47,25 @@ android {
         }
     }
 
+    signingConfigs {
+        create("release") {
+            val keyPropsFile = rootProject.file("key.properties")
+            if (keyPropsFile.exists()) {
+                val keyProps = java.util.Properties()
+                keyProps.load(java.io.FileInputStream(keyPropsFile))
+                keyAlias = keyProps["keyAlias"] as String?
```

detekt (1.23.8)

[warning] Use the safe cast ('as? String') instead of 'as String?'.

(detekt.potential-bugs.CastToNullableType)